src/Illuminate/Mail/MailServiceProvider.php

```diff
@@ -24,28 +24,44 @@ class MailServiceProvider extends ServiceProvider
      */
     public function register()
     {
+        $this->registerConnectionManager();
+
         $this->registerSwiftMailer();
 
+        $this->registerSwiftTransport();
+
         $this->registerIlluminateMailer();
 
         $this->registerMarkdownRenderer();
     }
 
+    /**
+     * Register the Illuminate mailer connection manager.
+     *
```

Comment:
Please add phpdoc
```PHP 
@see MailManager
```

Reply:
The framework uses `@see` only at class level.